libs/pages/logs/environment/src/lib/feature/deployment-logs-feature/deployment-logs-feature.tsx

```diff
@@ -57,26 +57,83 @@ export function getServiceStatuesById(services?: DeploymentStageWithServicesStat
   return null
 }
 
-export function DeploymentLogsFeature(props: DeploymentLogsFeatureProps) {
-  const { logs, loadingStatus, pauseStatusLogs, setPauseStatusLogs, statusStages } = props
-  const { serviceId = '' } = useParams()
-  const { stageId, updateServiceId } = useContext(ServiceStageIdsContext)
+export function DeploymentLogsFeature({ environment, statusStages }: DeploymentLogsFeatureProps) {
+  const { organizationId = '', projectId = '', environmentId = '', serviceId = '', versionId = '' } = useParams()
+  const { stageId } = useContext(ServiceStageIdsContext)
 
   const application = useSelector<RootState, ApplicationEntity | undefined>((state) =>
     selectApplicationById(state, serviceId)
   )
   const database = useSelector<RootState, DatabaseEntity | undefined>((state) => selectDatabaseById(state, serviceId))
+  const { data: dataDeploymentHistory } = useEnvironmentDeploymentHistory(projectId, environmentId)
+
+  const [logs, setLogs] = useState<EnvironmentLogs[]>([])
+  const [loadingStatusDeploymentLogs, setLoadingStatusDeploymentLogs] = useState<LoadingStatus>('not loaded')
+  const [messageChunks, setMessageChunks] = useState<EnvironmentLogs[][]>([])
+  const [debounceTime, setDebounceTime] = useState(1000)
+  const [pauseStatusLogs, setPauseStatusLogs] = useState<boolean>(false)
 
   useDocumentTitle(
-    `Deployment logs ${loadingStatus === 'loaded' ? `- ${application?.name || database?.name}` : '- Loading...'}`
+    `Deployment logs ${
+      loadingStatusDeploymentLogs === 'loaded' ? `- ${application?.name || database?.name}` : '- Loading...'
+    }`
   )
 
-  const hideDeploymentLogsBoolean = !(getServiceStatuesById(statusStages, serviceId) as Status)?.is_part_last_deployment
+  const { getAccessTokenSilently } = useAuth()
+  const chunkSize = 500
+
+  const deploymentLogsUrl: () => Promise<string> = useCallback(async () => {
+    // reset current Logs
+    setLogs([])
+
+    const url = `wss://ws.qovery.com/deployment/logs?organization=${organizationId}&cluster=${environment?.cluster_id}&project=${projectId}&environment=${environmentId}&version=${versionId}`
+    const token = await getAccessTokenSilently()
+
+    return new Promise((resolve) => {
+      environment?.cluster_id && resolve(url + `&bearer_token=${token}`)
+    })
+  }, [organizationId, environment?.cluster_id, projectId, environmentId, versionId, getAccessTokenSilently])
+
+  useWebSocket(deploymentLogsUrl, {
```

Comment:
We should use common hook based [this hook](https://github.com/Qovery/console/pull/744)